lib/utils/CSVDatabase.h

```diff
@@ -47,7 +47,7 @@ class CSVDatabase : public Database
     virtual
```

Comment:
It is good to add `override`, and `virtual` can be removed from all the member functions in this derived class, as well as other classes. If you prefer to make a future PR for that, that is fine with me.